data/json/items/tool_armor.json

```diff
@@ -1801,9 +1811,7 @@
     "turns_per_charge": 30,
     "revert_to": "mask_h20survivor",
     "use_action": { "type": "transform", "menu_text": "Turn off", "msg": "Your %s deactivates.", "target": "mask_h20survivor" },
-    "covers": [ "eyes", "mouth" ],
-    "environmental_protection": 16,
-    "encumbrance": 20
```

Comment:
i believe it was intended for this thing to have lower encumbrance when turned on

Reply:
What do you mean? I did,'t change the values, only the name of the bodypart_id

Reply:
You did remove encumbrance value of "on" version of divemask. And before it had 20 enc. when on and 30 when "off" (have "varsize" in mind too). Now "on" version will have the same enc. and better env. protection.

Reply:
Look at the changes again I have only edited the id. The encumbrance thing has been edited by another PR and it showed here when I rebased this against master.